<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>641</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

background: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>qt_resources/1/16.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>1024</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

background: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>qt_resources/1/9.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>986</x>
      <y>16</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

background: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>qt_resources/1/15.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>947</x>
      <y>15</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

background: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>qt_resources/1/14.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>909</x>
      <y>15</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

background: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>qt_resources/1/13.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>871</x>
      <y>15</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

background: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>qt_resources/1/12.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>835</x>
      <y>15</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

background: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>qt_resources/1/11.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>796</x>
      <y>15</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

background: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>qt_resources/1/5.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>13</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 16pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);

background: transparent;
</string>
    </property>
    <property name="text">
     <string>2:30:53 - PM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>423</x>
      <y>550</y>
      <width>181</width>
      <height>45</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">




QPushButton{
background: transparent;




/*border: 2px solid white;*/
border-radius: 12px;

}



QPushButton:pressed{

	background-color: rgb(222, 221, 218);
border-radius: 12px;

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/pi/zen_OS/qt_resources/APP_MENU/14.png</normaloff>:/pi/zen_OS/qt_resources/APP_MENU/14.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>601</y>
      <width>1031</width>
      <height>213</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_Phone">
     <property name="geometry">
      <rect>
       <x>53</x>
       <y>75</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">




QPushButton{
background: transparent;




/*border: 2px solid white;*/
border-radius: 12px;

}



QPushButton:pressed{

	background-color: rgb(222, 221, 218);
border-radius: 12px;

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/pi/zen_OS/qt_resources/APP_MENU/8.png</normaloff>:/pi/zen_OS/qt_resources/APP_MENU/8.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>135</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>180</y>
       <width>67</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Route">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>75</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">




QPushButton{
background: transparent;




/*border: 2px solid white;*/
border-radius: 12px;

}



QPushButton:pressed{

	background-color: rgb(222, 221, 218);
border-radius: 12px;

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/pi/zen_OS/qt_resources/APP_MENU/23.png</normaloff>:/pi/zen_OS/qt_resources/APP_MENU/23.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>135</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>75</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">




QPushButton{
background: transparent;




/*border: 2px solid white;*/
border-radius: 12px;

}



QPushButton:pressed{

	background-color: rgb(222, 221, 218);
border-radius: 12px;

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/pi/zen_OS/qt_resources/APP_MENU/21.png</normaloff>:/pi/zen_OS/qt_resources/APP_MENU/21.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>135</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>183</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>More</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>868</x>
       <y>75</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">




QPushButton{
	background: transparent;




/*border: 2px solid white;*/
border-radius: 12px;

}



QPushButton:pressed{

	background-color: rgb(222, 221, 218);
border-radius: 12px;

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/pi/zen_OS/qt_resources/APP_MENU/24.png</normaloff>:/pi/zen_OS/qt_resources/APP_MENU/24.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>135</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>678</x>
       <y>75</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">




QPushButton{
background: transparent;




/*border: 2px solid white;*/
border-radius: 12px;

}



QPushButton:pressed{

	background-color: rgb(222, 221, 218);
border-radius: 12px;

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/pi/zen_OS/qt_resources/APP_MENU/22.png</normaloff>:/pi/zen_OS/qt_resources/APP_MENU/22.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>135</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>78</x>
       <y>183</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>Phone</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>703</x>
       <y>183</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>Health</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1024</width>
       <height>213</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/pi/zen_OS/qt_resources/APP_MENU/2.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>183</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>Route</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>10</y>
       <width>181</width>
       <height>45</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">



QPushButton{
	background: transparent;




/*border: 2px solid white;*/
border-radius: 12px;

}



QPushButton:pressed{

	background-color: rgb(222, 221, 218);
border-radius: 12px;

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/pi/zen_OS/qt_resources/APP_MENU/13.png</normaloff>:/pi/zen_OS/qt_resources/APP_MENU/13.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>45</width>
       <height>45</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label_20</zorder>
    <zorder>pushButton_Phone</zorder>
    <zorder>label_13</zorder>
    <zorder>pushButton_Route</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>label_17</zorder>
    <zorder>pushButton_5</zorder>
    <zorder>pushButton_4</zorder>
    <zorder>label_18</zorder>
    <zorder>label_19</zorder>
    <zorder>label_21</zorder>
    <zorder>pushButton_2</zorder>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>-22</x>
      <y>547</y>
      <width>111</width>
      <height>45</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">




QPushButton{
background: transparent;




/*border: 2px solid white;*/
border-radius: 12px;

}



QPushButton:pressed{

	background-color: rgb(222, 221, 218);
border-radius: 12px;

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/pi/zen_OS/qt_resources/APP_MENU/16.png</normaloff>:/pi/zen_OS/qt_resources/APP_MENU/16.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>939</x>
      <y>548</y>
      <width>101</width>
      <height>45</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">




QPushButton{
background: transparent;




/*border: 2px solid white;*/
border-radius: 12px;

}



QPushButton:pressed{

	background-color: rgb(222, 221, 218);
border-radius: 12px;

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/pi/zen_OS/qt_resources/APP_MENU/15.png</normaloff>:/pi/zen_OS/qt_resources/APP_MENU/15.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame_camera_set_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>600</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="ClickableLabel" name="label_cam2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>506</width>
       <height>295</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>qt_resources/1/18.png</pixmap>
     </property>
    </widget>
    <widget class="ClickableLabel" name="label_cam1">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>0</y>
       <width>506</width>
       <height>295</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>qt_resources/1/18.png</pixmap>
     </property>
    </widget>
    <widget class="ClickableLabel" name="label_cam4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>310</y>
       <width>506</width>
       <height>295</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>qt_resources/1/18.png</pixmap>
     </property>
    </widget>
    <widget class="ClickableLabel" name="label_cam3">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>310</y>
       <width>506</width>
       <height>295</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>qt_resources/1/18.png</pixmap>
     </property>
    </widget>
    <zorder>label_cam4</zorder>
    <zorder>label_cam3</zorder>
    <zorder>label_cam1</zorder>
    <zorder>label_cam2</zorder>
   </widget>
   <widget class="QFrame" name="frame_camera_set_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>600</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="ClickableLabel" name="label_cam5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>506</width>
       <height>295</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>qt_resources/1/18.png</pixmap>
     </property>
    </widget>
    <widget class="ClickableLabel" name="label_cam6">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>0</y>
       <width>506</width>
       <height>295</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>qt_resources/1/18.png</pixmap>
     </property>
    </widget>
    <widget class="ClickableLabel" name="label_cam7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>310</y>
       <width>506</width>
       <height>295</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>qt_resources/1/18.png</pixmap>
     </property>
    </widget>
    <widget class="ClickableLabel" name="label_cam8">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>310</y>
       <width>506</width>
       <height>295</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>qt_resources/1/18.png</pixmap>
     </property>
    </widget>
   </widget>
   <zorder>frame_camera_set_2</zorder>
   <zorder>frame_camera_set_1</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>label_12</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_16</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>frame</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
